File: led.py
import RPi.GPIO as GPIO
from time import sleep

GPIO.setmode(GPIO.BCM)

blinkCount = 3
count = 0
yellow = 22
green = 17
red = 12
blue = 21

# Setup the pin that the LED is connected to
GPIO.setup(yellow, GPIO.OUT)
GPIO.setup(green, GPIO.OUT)
GPIO.setup(red, GPIO.OUT)
GPIO.setup(blue, GPIO.OUT)
try:
        while count < blinkCount:
                GPIO.output(yellow, True)
                GPIO.output(green, False)
		GPIO.output(red, True)
		GPIO.output(blue, False)
                print("GROUP 1 ON")
                sleep(2)
                GPIO.output(yellow, False)
                GPIO.output(green, True)
		GPIO.output(red, False)
		GPIO.output(blue, True)
                print("Group 2 ON")
                sleep(2)
                count += 1

finally:
        GPIO.cleanup()

File: LED.java
public class LED {
	public static void main(String[] args) {
		int blinkCount = 3;
		int count = 0;
		boolean yellow, blue, green, red;
		
		while(count < blinkCount) {
			yellow = true;
			green = false;
			red = false;
			blue = true;
			System.out.println("GROUP 1 ON");
			yellow = false;
			green = true;
			red = true;
			blue = false;
			System.out.println("GROUP 2 ON");
			yellow = false;
			green = false;
			red = false;
			blue = false;
		}
	}
	
}
